sklearn/neighbors/tests/test_lof.py

```diff
@@ -71,32 +78,32 @@ def test_lof_performance():
     assert roc_auc_score(y_test, y_pred) > 0.99
```

Comment:
We should be testing somewhere that `y_pred` is preserving dtype also.

Reply:
I understand your last message as indicating that we are not needing and assertion for `dtype` preservation because this is done in `test_score_samples`.